controlplane/kubeadm/internal/etcd/etcd_test.go

```diff
@@ -49,7 +49,7 @@ func TestEtcdMembers_WithErrors(t *testing.T) {
 		ErrorResponse:        errors.New("something went wrong"),
 	}
 
-	client, err := newEtcdClient(ctx, fakeEtcdClient)
+	client, err := newEtcdClient(ctx, fakeEtcdClient, 0)
```

Comment:
nit: let's use DefaultCallTimeout here. Just a bit more realistic and to avoid using 0 as a timeout

Reply:
Done!

Reply:
Thx!